scripts/keras_classification.py

```diff
@@ -0,0 +1,738 @@
+                        The root path to where the dataset is stored
+  --dataset-kwargs DATASET_KWARGS
+                        kew word arguments to be passed to dataset
+                        constructor, should be given as a json object
+  --model-tag MODEL_TAG
+                        A tag to use for the model for saving results under
+                        save-dir, defaults to the model arch and dataset used
+  --save-dir SAVE_DIR   The path to the directory for saving results
+  --train-dataset-parallel-calls DATASET_PARALLEL_CALLS
+                        the number of parallel workers for train dataset loading
+  --train-shuffle-buffer-size SHUFFLE_BUFFER_SIZE
+                        Shuffle buffer size for train dataset loading
+  --train-prefetch-buffer-size PREFETCH_BUFFER_SIZE
+                        Prefetch buffer size for train dataset loading
+  --val-dataset-parallel-calls DATASET_PARALLEL_CALLS
+                        the number of parallel workers for val dataset loading
+  --val-shuffle-buffer-size SHUFFLE_BUFFER_SIZE
+                        Shuffle buffer size for val dataset loading
+  --val-prefetch-buffer-size PREFETCH_BUFFER_SIZE
+                        Prefetch buffer size for val dataset loading
+  --recipe-path RECIPE_PATH
+                        The path to the yaml file containing the modifiers and
+                        schedule to apply them with. If set to
+                        'transfer_learning', then will create a schedule to
+                        enable sparse transfer learning. Recipe from SparseZoo
+                        could be specified using model stub starting with "zoo:"
+  --sparse-transfer-learn
+                        Enable sparse transfer learning modifiers to enforce
+                        the sparsity for already sparse layers. The modifiers
+                        are added to the ones to be loaded from the recipe-
+                        path
+  --eval-mode           Puts into evaluation mode so that the model can be
+                        evaluated on the desired dataset
+  --train-batch-size TRAIN_BATCH_SIZE
+                        The batch size to use while training
+  --test-batch-size TEST_BATCH_SIZE
+                        The batch size to use while testing
+  --logs-dir LOGS_DIR   The path to the directory for saving logs
+  --save-best-after SAVE_BEST_AFTER
+                        start saving the best validation result after the
+                        given epoch completes until the end of training
+  --save-epochs SAVE_EPOCHS [SAVE_EPOCHS ...]
+                        epochs to save checkpoints at
+  --init-lr INIT_LR     The initial learning rate to use while training, the
+                        actual initial value used should be set by the
+                        sparseml recipe
+  --optim-args OPTIM_ARGS
+                        Additional args to be passed to the optimizer passed
+                        in as a json object
+#############
```

Comment:
For this help doc, please match the style setup in pytorch_vision.py and other scripts. Specifically the use of spaces and the use of Example command rather than Example: